### PR TITLE
feat: add isNonEmptyString

### PR DESCRIPTION
**Pull request by guillaumearm**
Ref #40

### CODE REVIEW

**Comment by guillaumearm** [comment]
@char0n I'm a bit confused to name the PR.
Should i name it : 
`feat: add isNonEmptyString`
or
`feat(#40): add isNonEmptyString`
?

**Comment by char0n** [comment]
Should be 
```
feat: add isNonEmpty (#id_of_pull_requst)

Closes|Ref #id_of_the issue
```

See https://github.com/char0n/ramda-adjunct/blob/master/CONTRIBUTING.md#-commit-message-guidelines

I am always re-checking and ammending commit messages during shash/rebase of PRs.

File: src/isNonEmptyString.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {Boolean}
+ * @see {@link RA.isEmptyString|isEmptyString}
+ * @example
+ *
+ * RA.isNonEmptyString('42'); // => true
+ * RA.isNonEmptyString(''); // => false
+ * RA.isNonEmptyString(new String('42')); // => false
+ * RA.isNonEmptyString(new String()); // => false
```
Comment (char0n) [inline src/isNonEmptyString.js]: replace with `new String('')` just to be explicit

File: test/isNonEmptyString.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import eq from './shared/eq';
+
+
+describe('isNonEmptyString', function () {
+  it('tests a value to be an empty `String`', function () {
+    eq(RA.isNonEmptyString('42'), true);
+
+    eq(RA.isNonEmptyString(''), false);
+    eq(RA.isNonEmptyString(new String('42')), false);
+    eq(RA.isNonEmptyString(new String()), false);
```
Comment (char0n) [inline test/isNonEmptyString.js]: replace with `new String('')` just to be explicit

**Comment by guillaumearm** [comment]
what is the deal with permissions in circleci ?

**Comment by char0n** [comment]
Haven't been able to find out yet. Seems like maybe betternpmrun related
problem.

**Comment by char0n** [comment]
Nice job

**Pull request merged**